paper/paper.md

```diff
@@ -0,0 +1,193 @@
+Furthermore, @bland2009 called for a focus on the width of confidence intervals 
+rather than the power of the test in sample size calculations. 
+There are many software packages
+for hypothesis-based sample size calculation, such as [Stata](https://www.stata.com/),
+[PASS](https://www.ncss.com/software/pass/), and [G*Power](https://www.gpower.hhu.de).
+However, many research projects aim at estimation rather than hypothesis testing,
+sample size calculation approaches for which are largely missing from other software
+packages. 
+We therefore developed a comprehensive tool for precision-based sample size calculation.
+
+# Development
+
+We programmed a package in the R programming language that offers sample size
+calculation for estimation-based research. We implemented the most common
+measures used in descriptive research; see Table 1 for a list of functions,
+corresponding measures, and methods used for calculation. 
+There are two approaches for each measure. First, based on a given sample size,
+e.g. for a retrospective data analysis, the precision of an expected measure can
+be calculated. Precision is expressed as the confidence interval around the
+measure. The level of confidence can be specified; the usual 95%-confidence
+interval (CI) is the default. Second, based on a given precision (i.e. CI), the sample size can be calculated. This is mainly of use in the
+planning of prospective studies, where the aim is to estimate a measure of
+interest with enough confidence. The available functions in the package require
+common input arguments; either the sample size or the width of the CI, and the level of the CI. Depending on the function,
+further specific input arguments are required such as the expected area under
+the curve (AUC) for test accuracy.
+
+
+Table 1. Measures, functions and methods provided in presize.
+
+Measure | Function | Methods available 
+-------- | ---------- | --------
+**Descriptive measures** | |
+Mean | `prec_mean` |
+Proportion | `prec_prop` | Wilson, Agresti-Coull, exact, Wald [see @brown2001]
+Rate | `prec_rate` | Score, variance stabilizing, exact, Wald [see @barker2002]
+**Absolute differences** | |
+Mean difference | `prec_meandiff` |
+Risk difference | `prec_riskdiff` | Newcombe [@newcombe1998], Miettinen-Nurminen [@mn1985], Agresti-Caffo [@ac2000], Wald
+**Relative differences** | |
+Odds ratio | `prec_or` | Gart, Wolff, independence smoothed logit [see @fll2015]
+Risk ratio | `prec_riskratio` | Koopman [@koopman1984], Katz [@kbap1978]
+Rate ratio | `prec_rateratio` | Rothman [@rg2018]
+**Correlation measures** | |
+Correlation coefficient | `prec_cor` | Pearson, Kendall, Spearman [see @bw2000]
+Intraclass correlation | `prec_icc` | @bonnett2002
+Limit of agreement | `prec_lim_agree` | @ba1986
+Cohen's kappa | `prec_kappa` | @rd2012
+**Diagnostic measures** | |
+Sensitivity | `prec_sens` | As per `prec_prop`
+Specificity | `prec_spec` | As per `prec_prop`
+Area under the curve | `prec_auc` | @hm1982
```

Comment:
Add positive and negative likelihood ratios.

Reply:
thanks